### DISCUSSION TITLE
Applying specific photography style with the correct lightning and angle? possible with Fooocus or ComfyUI workflow required?

### DISCUSSION THREAD

**Discussion by chienb** [lllyasviel/Fooocus#3194] [Q&A] [answered]
I'm trying to manipulate a selfie by applying a specific style that closely resembles Mark Hunter's style. [https://www.thecobrasnake.com/](url)
![image](https://github.com/lllyasviel/Fooocus/assets/12900266/325fcd5a-2411-44f6-9f9a-e0a91e2565bb)
Variation image

![image](https://github.com/lllyasviel/Fooocus/assets/12900266/25c00a57-d9f3-43d5-b2a8-e330cb9fbe24)
subject selfie (image prompt)

So I'm wondering if Fooocus able to do the following: 

1. manipulate a selfie image and apply the correct photography style, lightning and angle of the subject
2. crop image to only show headshot, remove background

Using the image above as reference for the style and this selfie, I was able generate something like this by mixing inpainting and variation settings. I think this is a good start, but losing a lot of resemblance of the subject.

![image](https://github.com/lllyasviel/Fooocus/assets/12900266/741ca990-ed90-4498-9ce9-e52f4179eb60)

Would this be something more suitable for Comfyui since we are dealing with workflows?

**Comment by wisdombox44** [comment]
You can remove background with any photo editor, than you can try with fooocus image prompt feature.
This may help.
https://www.youtube.com/watch?v=sN1l79RheSw
for more advanced feature Have a look this channel
https://www.youtube.com/watch?v=czcgJnoDVd4&t=67s

**Comment by mashb1t** [comment, marked as answer]
@chienb please check out https://github.com/lllyasviel/IC-Light, which basically does exactly what i believe you want!
